API/Program.cs

```diff
@@ -158,7 +158,8 @@
     {
         context.HttpContext.Response.StatusCode = StatusCodes.Status429TooManyRequests;
         const string readMore =
-            "Read more about our rate limits at https://github.com/osu-tournament-rating/otr-wiki/blob/master/api/limits/en.md";
+            // TODO:
```

Comment:
```suggestion
            // TODO: Link to the API Terms of Service document
```
Sorry should've specified LOL but these todo tags are indexed by Rider and show up with whatever comment follows them. Makes for easy notes so make sure you fill them in